src/tasks/python/PythonExtensionHelper.ts

```diff
@@ -19,6 +17,23 @@ export namespace PythonExtensionHelper {
     }
 
     export async function getLauncherFolderPath(): Promise<string> {
+        const pyExt = await getPythonExtension();
+
+        // tslint:disable:no-unsafe-any no-any
```

Comment:
Can you install eslint instead of tslint? The suppressions are different for that as well.